publish-frontend/src/components/post-form.jsx

```diff
@@ -34,10 +34,12 @@ import { createTag } from '@/lib/tags';
 const PostForm = ({ tags, user, authors, post }) => {
   const toast = useToast();
   const router = useRouter();
+
   const hasCreatedPost = useRef(false);
 
   const [title, setTitle] = useState('(UNTITLED)');
   const [isEditingTitle, setIsEditingTitle] = useState(false);
+  const [postId, setPostId] = useState(post?.id ? post.id : null);
```

Comment:
```suggestion
  const [postId, setPostId] = useState(post?.id);
```
It doesn't look like we need to differentiate between `undefined` and `null` here.